test/unit/workers/publishing_api_vanish_worker_test.rb

```diff
@@ -34,9 +33,8 @@ class PublishingApiVanishWorkerTest < ActiveSupport::TestCase
       },
     )
 
-    PublishingApiVanishWorker.new.perform(
-      publication.document.content_id, "en", discard_drafts: true
-    )
+    PublishingApiVanishWorker.perform_async(publication.content_id, "en", true)
```

Comment:
This should be:

```suggestion
    PublishingApiVanishWorker.perform_async(publication.document.content_id, "en", true)
```

I don't think it makes a difference, mind.

Reply:
You can fetch the content_id directly from the edition (publication) so it seems superfluous to explicitly call the document. However your point highlighted that the test file wasn't consistent which has now been addressed, good spot!